ImageData/FileLoader.h

```diff
@@ -128,54 +127,65 @@ class FileLoader {
 
     // check for mirlib (MIRIAD) error; returns true for other image types
     virtual bool CanOpenFile(std::string& error);
+    // Do anything required to open the file at given HDU (set up cache size, Image object)
+    virtual void OpenFile(const std::string& hdu) = 0;
 
-    // get shape and axis information from image data and coordinate system
-    bool FindShape(IPos& shape, int& spectral_axis, int& stokes_axis, std::string& message);
-    void FindCoordinates(int& spectral_axis, int& stokes_axis);
+    // Return the opened casacore image
+    virtual ImageRef GetLoaderImage() = 0;
```

Comment:
I think renaming this is definitely a good idea, but I would remove the `Loader` and just call it `GetImage` -- it seems redundant given that it will be called on an object which will probably be called `_loader`, and this is consistent with `GetSlice`, etc..

Reply:
OK